clang/include/clang-c/CXString.h

```diff
@@ -14,6 +14,7 @@
 #ifndef LLVM_CLANG_C_CXSTRING_H
 #define LLVM_CLANG_C_CXSTRING_H
 
+#include <stddef.h>
```

Comment:
Better to put this at the end of the include list; that leaves less chance for those headers to accidentally transitively include stddef.h without including it themselves.